src/main/kotlin/com/autonomousapps/DependencyAnalysisExtension.kt

```diff
@@ -63,6 +64,10 @@ open class DependencyAnalysisExtension(objects: ObjectFactory) {
   fun issues(action: Action<IssueHandler>) {
     action.execute(issueHandler)
   }
+
+  var dependencyRenamingMap = objects.mapProperty(String::class.java, String::class.java).also {
```

Comment:
should be `val`.

To your question about ease of use, Groovy DSL users (who are more than 95% of all gradle users), can do
```groovy
dependencyAnalysis {
  dependencyRenamingMap = [...]
}
```
(Gradle magic makes that work)
Kotlin DSL users must do:
```kotlin
dependencyAnalysis {
  dependencyRenamingMap.putAll(mapOf(...))
}
```
I think that's fine.

Reply:
I find the `putAll` not so elegant. It looks like there is a state already and you want to add new content. 

Do you mind if I would keep the var ?